app/models/cohort.rb

```diff
@@ -1,51 +1,64 @@
 # frozen_string_literal: true
 
 class Cohort < ApplicationRecord
-  has_many :schedules, class_name: "Finance::Schedule"
-  has_many :partnerships
-  has_many :statements
   has_many :call_off_contracts
   has_many :npq_contracts
+  has_many :partnerships
+  has_many :schedules, class_name: "Finance::Schedule"
+  has_many :statements
+
+  # Class Methods
+
+  # Find the cohort with start year matching the given year.
+  def self.[](year)
+    find_by_start_year(year)
+  end
+
+  def self.active_registration_cohort
+    where(registration_start_date: ..Date.current).order(start_year: :desc).first
+  end
 
   def self.current
-    # TODO: Register and Partner 262: Figure out how to update current year
-    if FeatureFlag.active?(:multiple_cohorts)
-      where("academic_year_start_date <= ?", Time.zone.now).order(start_year: :desc).first
-    else
-      find_by(start_year: 2021)
-    end
+    date_range = (Date.current - 1.year + 1.day)..Date.current
+    where(academic_year_start_date: date_range).first
```

Comment:
Im a bit unsure around our rules for the start/end of a cohort, and how `registration_start_date` and `academic_year_start_date` differ, also if this is similar for both NPQ and ECF, do you mind clarifying?

Reply:
For a cohort, I am considering:
- the academic year start date as the start of the cohort. 
- the registration start date, earlier than the academic year start date, and meant to allow the registration of participants for that cohort.

Reply:
The rules we followed when this was originally added were (if we needed to decide on which cohort to show in the UI for example) the `active_registration_cohort` would normally be the `current` except for the period between the `registration_start_date` and the start of the `next` cohort.